scripts/CEE/restart-dns-default/README.md

```diff
@@ -0,0 +1,14 @@
+# Restart default dns pods in openshift-dns namespace
+
+## Desctiption
+ 
+This script will trigger a rollout of daemonset, controlling dns default pods. Such action might be required when a customer is not able to perform it themself due to not having enough permissions etc, but dns-default pods need to be restarted.  
+
+## Usage
+
+```bash
+ocm backplane managedjob create CEE/restart-dns-default -p NAMESPACE=openshift-dns
```

Comment:
I assume the script is solely intended for restarting dns pods within the `openshift-dns`, is that necessary to specify it as a parameter `-p NAMESPACE` or just hard-code in the script?

Reply:
For this particular script we definitely can have it hard-coded. The idea was if this script is considered as a valid option for Managed Scripts and proven working then to introduce/or develop more sophisticated script  that will target different `openshift-* `namespaces e.g. `openshift-kube-apiserver` or `openshift-ovn-kubernetes` and thats where we pass a namespace as a parameter `-p NAMESPACE. ` I will follow your suggestion and hard code `var` in the script